envoy/BAZEL_VERSION

```diff
@@ -1 +1 @@
-0.10.1
\ No newline at end of file
+0.11.0
```

Comment:
I think you need to remove the trailing `\n` (see old file)

Reply:
Good catch. Fixed.